model/telemetry/metrics.yaml

```diff
@@ -0,0 +1,76 @@
+groups:
+  - id: metric.telemetry.sdk.trace.spans.ended
+    type: metric
+    metric_name: telemetry.sdk.trace.spans.ended
+    stability: experimental
+    brief: "The number of spans which have been ended"
+    instrument: counter
+    unit: "1"
+
+  - id: metric.telemetry.sdk.trace.spans.sampled
+    type: metric
+    metric_name: telemetry.sdk.trace.spans.sampled
+    stability: experimental
+    brief: "The number of spans which have been ended AND are sampled"
+    instrument: counter
+    unit: "1"
+
+  - id: metric.telemetry.sdk.trace.processor.queue_size
+    type: metric
+    metric_name: telemetry.sdk.trace.processor.queue_size
+    stability: experimental
+    brief: "The number of spans in the queue of an instance of an SDK span processor"
+    note: |
+      Only applies to span processors which use a queue, e.g. the SDK Batching Span Processor.
+    instrument: updowncounter
+    unit: "1"
+    attributes:
+      - ref: telemetry.sdk.processor.type
+      - ref: telemetry.sdk.component.id
+
+  - id: metric.telemetry.sdk.trace.processor.total
+    type: metric
+    metric_name: telemetry.sdk.trace.processor.total
+    stability: experimental
+    brief: "The total number of spans submitted for processing to this span processor"
+    instrument: counter
+    unit: "1"
+    attributes:
+      - ref: telemetry.sdk.processor.type
+      - ref: telemetry.sdk.component.id
+
+  - id: metric.telemetry.sdk.trace.processor.failed
+    type: metric
+    metric_name: telemetry.sdk.trace.processor.failed
+    stability: experimental
+    brief: "The total number of spans which could not be successfully processed"
+    instrument: counter
+    unit: "1"
+    attributes:
+      - ref: telemetry.sdk.processor.type
+      - ref: telemetry.sdk.component.id
+      - ref: error.type
+        examples: ["queue full"]
+
+  - id: metric.telemetry.sdk.trace.exporter.total
```

Comment:
([see here](https://opentelemetry.io/docs/specs/semconv/general/metrics/#do-not-use-total)) total should not be used as a metric name suffix, as it might be confusing.

Maybe something like:

`telemetry.sdk.trace.exporter.spans_submitted`

Reply:
Fixed in https://github.com/open-telemetry/semantic-conventions/commit/42489175c14991a51f29b395a269b0ac3ecf9454